app/controllers/app_menu_list.js

```diff
@@ -51,8 +51,11 @@ export default Ember.Controller.extend(storageType, OrganisationMixin, {
         queryParams: { codeId: code.get("id"), storageType: storageTypeName }
       });
     },
+    createInternationalOrder() {
```

Comment:
doesnt sound right 🤔 

The method name is createInternationalOrder,
but its just changing the route